<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UusiMaksuDlg</class>
 <widget class="QDialog" name="UusiMaksuDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uusi maksu</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="pankkiLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Saajan tili (IBAN)</string>
       </property>
       <property name="buddy">
        <cstring>ibanEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KpIbanEdit" name="ibanEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Saajan nimi</string>
       </property>
       <property name="buddy">
        <cstring>saajaEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="saajaEdit">
       <property name="maxLength">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maksupäivä</string>
       </property>
       <property name="buddy">
        <cstring>pvmEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KpDateEdit" name="pvmEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Määrä</string>
       </property>
       <property name="buddy">
        <cstring>euroEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="KpEuroEdit" name="euroEdit">
       <property name="EiMiinus" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QRadioButton" name="viiteRadio">
       <property name="text">
        <string>Viite</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="KpViiteEdit" name="viiteEdit">
       <property name="maxLength">
        <number>34</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QRadioButton" name="viestiRadio">
       <property name="text">
        <string>Viesti</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="viestiEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maxLength">
        <number>120</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Laskun numero</string>
       </property>
       <property name="buddy">
        <cstring>laskuNumeroEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="laskuNumeroEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KpDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpdateedit.h</header>
  </customwidget>
  <customwidget>
   <class>KpEuroEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpeuroedit.h</header>
  </customwidget>
  <customwidget>
   <class>KpViiteEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">tools/kpviiteedit.h</header>
  </customwidget>
  <customwidget>
   <class>KpIbanEdit</class>
   <extends>QLineEdit</extends>
   <header>tools/kpibanedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ibanEdit</tabstop>
  <tabstop>saajaEdit</tabstop>
  <tabstop>pvmEdit</tabstop>
  <tabstop>euroEdit</tabstop>
  <tabstop>viiteRadio</tabstop>
  <tabstop>viiteEdit</tabstop>
  <tabstop>viestiRadio</tabstop>
  <tabstop>viestiEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UusiMaksuDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UusiMaksuDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viiteRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>viiteEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viestiRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>viestiEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
